docs/pages/versions/v41.0.0/sdk/notifications.md

```diff
@@ -34,6 +34,27 @@ The **`expo-notifications`** provides an API to fetch push notification tokens a
 
 <InstallSection packageName="expo-notifications" />
 
+### Optional setup
```

Comment:
Maybe take some inspiration from what I did here https://github.com/react-native-voice/voice/blob/2ee412a19aeaa1b4d4f03ff8b5599f034a286edf/README.md#config-plugin ([code](https://github.com/react-native-voice/voice/pull/317/files)). Note the links to prebuilding and the plugins array, etc.